app/views/applications/show.html.erb

```diff
@@ -5,8 +5,27 @@
 <p>State: <%= @application.state %></p>
 <p>Zip Code: <%= @application.zip %></p>
 <p>Description: <%= @application.description %></p>
+
 <p>Applying for: </p>
 <% @pets.each do |pet| %>
 <%=link_to "#{pet.name}", "/pets/#{pet.id}" %></p>
-<% end %>
-<p>Status: <%= @application.status %></p>
\ No newline at end of file
+<% end %><br>
+
+<% if @application.status == "In Progress" %>
```

Comment:
Possible opportunity for refactor (just making a note for future reference): I think we could add a helper method in the model to check this that we could call here, which would give us the ability to test more things and move some of this logic out of our view.